src/test/java/org/broadinstitute/dsm/GBFTest.java

```diff
@@ -178,8 +178,9 @@ public void inputConfirmationGBFKit() throws Exception {
             GBFRequestUtil gbf = new GBFRequestUtil();
                 String query = "SELECT * " +
                         "FROM ddp_kit_request req  " +
-                        "LEFT JOIN ddp_kit kit ON  " +
-                        "(req.dsm_kit_request_id = kit.dsm_kit_request_id) " +
+                        "LEFT JOIN ddp_kit kit ON (req.dsm_kit_request_id = kit.dsm_kit_request_id)  " +
+                        "LEFT JOIN ddp_participant_exit ex ON (ex.ddp_instance_id = req.ddp_instance_id AND ex.ddp_participant_id = req.ddp_participant_id)   " +
```

Comment:
you don't need to join with `ddp_participant_exit` table if it is not used in the query